yt/utilities/lib/image_samplers.pxd

```diff
@@ -31,7 +30,7 @@ ctypedef void generate_vector_info_function(ImageSampler im,
             np.float64_t v_dir[3], np.float64_t v_pos[3]) nogil
 
 cdef struct ImageAccumulator:
-    np.float64_t rgba[Nch]
+    np.float64_t rgba[4]  # Nch = 4
```

Comment:
I mean, if we can't use `Nch` here then why have `Nch` above?  If we change `Nch` in the earlier code, this would then break, right?

Reply:
above where ? that's the only line that's using it in this file, and yeah unfortunately the refactor pattern that I used everywhere else doesn't work here

Reply:
Ah, I meant above this in the file view, which included stuff like octree raytracing, where it is defined with your trick.

Reply:
Looks like it's actually only used here and in `image_samplers.pyx`, I missed this. Let's clean this up then.